airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_error_reporter/ErrorReportingClient.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.scheduler.persistence.job_error_reporter;
+
+import io.airbyte.config.FailureReason;
+import io.airbyte.config.StandardWorkspace;
+import java.util.Map;
+
+/**
+ * A generic interface for a client that reports errors
+ */
+public interface ErrorReportingClient {
+
+  /**
+   * Report a job failure reason
+   */
+  void report(StandardWorkspace workspace, final FailureReason reason, final String dockerImage, Map<String, String> metadata);
```

Comment:
fwiw this interface is very specific for connector job failures which makes this maybe not a generic sentry client. I think this is completely fine if we rename this iface to `ConnectorJobErrorReportingClient` or something that clearly indicates it is not meant to be a generic error client.